tests/fakeGRPCServer/gRPCServer.ts

```diff
@@ -0,0 +1,14 @@
+import {startServer} from '@npmsoluto/soluto-kafka-grpc-target';
```

Comment:
I don't think you should use that package here. (again open-source vs private package)